src/settings/EditAliasesFormN.js

```diff
@@ -110,7 +110,7 @@ class _EditAliasesForm {
 				],
 				actionButtonAttrs: actionButtonAttrs
 			}
-		})
+		}).sort((a, b) => (a.cells[0] > b.cells[0]) ? 1 : -1)
```

Comment:
I would sort them here before "map" so that we sort aliases and not table cells.

Reply:
Thanks! I've changed this to sort them before "map".